src/test/e2e/bundle_test.go

```diff
@@ -624,3 +624,16 @@ func TestInvalidBundle(t *testing.T) {
 	stderr := createLocalError(bundleDir, e2e.Arch)
 	require.Contains(t, stderr, "unknown field")
 }
+
+func TestArchCheck(t *testing.T) {
```

Comment:
This test passes on my arm64 Mac, may need a bit more logic to ensure we're setting the proper arch depending on the system on which this test runs

Reply:
mmm, yeah i wondered if something like that was going to happen. Thanks @UncleGedd i'll fix it.

Reply:
interesting i saw that too in a previous CI run, but couldn't figure out where it was coming from so i re-ran the job and it went away.